ceph/ceph_admin/cephadm.py

```diff
@@ -0,0 +1,125 @@
+from utility.log import Log
+
+from .ceph import CephCLI
+
+log = Log(__name__)
+
+CEPH_CONFIG_DIR = "/etc/ceph"
+CEPH_LIB_DIR = "/var/lib/ceph"
+CEPHADM_LIB_DIR = "/var/lib/cephadm"
+
+
+class CephadmOpsFailure(Exception):
+    pass
+
+
+class Cephadm(CephCLI):
```

Comment:
This could lead to conflicts as the current design treats `class CephAdmin` as the interface to `cephadm` and I believe we are attempting to create a one to one mapping with the command `cephadm` with `class Cephadm`.

If this is a newer implementation... we can threat `Cephadm` as a separate object without having a parent class. This way we could align with our long term goal. WDYT?

That is
```
class Cephadm:
  """
    This object interfaces with the command cephadm and treats all sub commands as objects and actions as 
     methods
  """
```

Reply:
Yes, consolidating all the methods related to cephadm at same place.